src/libs/OptionsListUtils.ts

```diff
@@ -1569,43 +1498,36 @@ function getShareDestinationOptions(
 /**
  * Format personalDetails or userToInvite to be shown in the list
  *
- * @param {Object} member - personalDetails or userToInvite
- * @param {Object} config - keys to overwrite the default values
- * @returns {Object}
+ * @param member - personalDetails or userToInvite
+ * @param config - keys to overwrite the default values
  */
-function formatMemberForList(member, config = {}) {
+function formatMemberForList(member: ReportUtils.OptionData, config: ReportUtils.OptionData | EmptyObject = {}) {
     if (!member) {
         return undefined;
     }
 
-    const accountID = lodashGet(member, 'accountID', '');
+    const accountID = member.accountID;
 
     return {
-        text: lodashGet(member, 'text', '') || lodashGet(member, 'displayName', ''),
-        alternateText: lodashGet(member, 'alternateText', '') || lodashGet(member, 'login', ''),
-        keyForList: lodashGet(member, 'keyForList', '') || String(accountID),
+        text: member.text ?? member.displayName ?? '',
+        alternateText: member.alternateText ?? member.login ?? '',
+        keyForList: member.keyForList ?? String(accountID),
         isSelected: false,
         isDisabled: false,
         accountID,
-        login: lodashGet(member, 'login', ''),
+        login: member.login ?? '',
         rightElement: null,
-        icons: lodashGet(member, 'icons'),
-        pendingAction: lodashGet(member, 'pendingAction'),
+        icons: member.icons,
+        pendingAction: member.pendingAction,
         ...config,
     };
 }
 
 /**
  * Build the options for the Workspace Member Invite view
- *
- * @param {Object} personalDetails
- * @param {Array<String>} betas
- * @param {String} searchValue
- * @param {Array} excludeLogins
- * @returns {Object}
  */
-function getMemberInviteOptions(personalDetails, betas = [], searchValue = '', excludeLogins = []) {
-    return getOptions([], personalDetails, {
+function getMemberInviteOptions(personalDetails: OnyxCollection<PersonalDetails>, betas: Beta[] = [], searchValue = '', excludeLogins: string[] = []) {
```

Comment:
```suggestion
function getMemberInviteOptions(personalDetails: OnyxEntry<PersonalDetailsList>, betas: Beta[] = [], searchValue = '', excludeLogins: string[] = []) {
```

PersonalDetails is not a Onyx Collection.

Add return type.